core/lib/mcxtx.h

```diff
@@ -229,12 +229,18 @@
          * Windows system calls.  These fields are ignored for 32-bit processes
          * that are not WOW64, or if the underlying processor does not support
          * SSE.  Use dr_mcontext_xmm_fields_valid() to determine whether the
-         * fields are valid.
+         * fields are valid. Use dr_mcontext_zmm_fields_valid() to determine
+         * whether zmm registers are preserved.
          *
-         * When the fields are valid, on processors with AVX enabled (i.e.,
+         * When the xmm fields are valid, on processors with AVX enabled (i.e.,
          * proc_has_feature(FEATURE_AVX) returns true), these fields will
          * contain the full ymm register values; otherwise, the top 128
          * bits of each slot will be undefined.
+         *
+         * When the zmm fields are valid, it implies that proc_has_feature(FEATURE_AVX512)
```

Comment:
Make Doxygen links for both.

Reply:
I found only () + # works. I rewrote to "proc_has_feature() with #FEATURE_AVX512 ".